l2geth/rollup/sync_service.go

```diff
@@ -412,14 +412,12 @@ func (s *SyncService) SequencerLoop() {
 // compare against the transactions it has in its local state. The sequencer
 // should reorg based on the transaction batches that are posted because
 // L1 is the source of truth. The sequencer concurrently accepts user
-// transactions via the RPC.
+// transactions via the RPC. When reorg logic is enabled, this should
+// also call `syncBatchesToTip`
 func (s *SyncService) sequence() error {
 	if err := s.syncQueueToTip(); err != nil {
 		return fmt.Errorf("Sequencer cannot sequence queue: %w", err)
 	}
-	if err := s.syncBatchesToTip(); err != nil {
```

Comment:
Can we comment this out instead of removing it?

Reply:
No need to comment it out, that is what git is for, there is a clear comment above this function + a good comment in git